packages/unlock-js/src/abis.ts

```diff
@@ -1043,7 +1043,7 @@ const abis = {
         "function tokenOfOwnerByIndex(address _keyOwner,uint256 _index) view returns (uint256)",
```

Comment:
@julien51 we need this one? update after build of `unlock-js`

Reply:
Hum, let's remove from this PR please.